src/pages/CoinList/index.js

```diff
@@ -230,22 +227,9 @@ class CoinList extends React.Component {
             {sortedList.map((coin) => {
               return (
                 <CoinListItem
-                  key={coin.id}
-                  id={coin.id}
+                  key={uuidv4()}
```

Comment:
this will be randomly created every time and defeats the purpose of having a constant uniq key,  use the coin name as id